examples/spire/cluster3/kustomization.yaml

```diff
@@ -2,10 +2,9 @@
 apiVersion: kustomize.config.k8s.io/v1beta1
 kind: Kustomization
 
-namespace: spire
```

Comment:
Why do we need to remove this?

Reply:
It was required to separate resources into 2 different namespaces, but decided to use only "spire" namespace